src/camera.rs

```diff
@@ -45,35 +43,20 @@ fn setup_camera(mut commands: Commands) {
             input_map: MainCameraBundle::default_input_map(),
             action_state: ActionState::default(),
         },
-        pbr: PerspectiveCameraBundle {
+        perspective_camera: PerspectiveCameraBundle {
             transform: Transform::from_xyz(0.0, 30.0, 10.0).looking_at(Vec3::ZERO, Vec3::Y),
             ..Default::default()
         },
     });
 }
 
-struct PlayerCameraEvent(Vec3);
-
-fn track_player(
```

Comment:
I feel like having a single system which sole purpose is to "trigger" the code in another one is a bit redundant, just merge all of this into a single system. If it is needed, you can add the event reading later.